mhctools/base_commandline_predictor.py

```diff
@@ -21,7 +21,7 @@
 from six import string_types
 from typechecks import require_string, require_integer, require_iterable_of
 from mhcnames import normalize_allele_name
-from mhcnames.parsing_helpers import AlleleParseError
```

Comment:
@julia326 is this correctly using the old `mhcnames` and will work with your PR?